### ISSUE TITLE
First chapter - Title missmatch

### ISSUE THREAD

**Issue by friki67** [aedocw/epub2tts#223]
Hello. Last release. The app is using the title for the first part as the title for the book (tried in two books).
```
Author: Me myself
Title: My Title

# Part 1
```

is setting "Part 1" as the title in the audiobook file.

**Comment by aedocw** [comment]
The last two books I did seemed to work fine but I'll see if I can reproduce this.

**Comment by aedocw** [comment]
I tested with the following in "testbook.txt":

```
Author: Me myself
Title: My Title

# Part 1

This is the text from part one of this test book.

# Part 2

This is part two of this book.
```

and it looks good:

```
  Metadata:                                                                                                                                               
    ARTIST          : Me myself                                                                                                                           
    ALBUM           : My Title
    DESCRIPTION     : Made with https://github.com/aedocw/epub2tts
    encoder         : Lavf58.76.100
  Chapters:
    Chapter #0:0: start 0.000000, end 5.741000
      Metadata:
        title           : Part 1
    Chapter #0:1: start 5.741000, end 10.959000
      Metadata:
        title           : Part 2
```

Can you share the ffmpeg output you see at the end of one of the books you tried with?

**Comment by aedocw** [comment]
```
  Metadata:                                                                                                                                               
    ARTIST          : Eve Curie                                                                                                                           
    ALBUM           : Madame Curie                                                                                                                        
    DESCRIPTION     : Made with https://github.com/aedocw/epub2tts                                                                                        
    encoder         : Lavf58.76.100                                                                                                                       
  Chapters:                                                                                                                                               
    Chapter #0:0: start 0.000000, end 297.312000                                                                                                          
      Metadata:                                                                                                                                           
        title           : Introduction                                                                                                                    
    Chapter #0:1: start 297.312000, end 1974.480000                                                                                                       
      Metadata:                                                                                                                                           
        title           : CHAPTER 1 Manya                                                                                                                 
    Chapter #0:2: start 1974.480000, end 3559.656000
```

```
ᐅ pip show epub2tts
Name: epub2tts
Version: 2.4.0
```

**Comment by friki67** [comment]
Thank you.

Yes, I'm getting same metadata (using exiftool) and "album" contains the title of the book. I don't know why books created with former version show me in my book player the right title and the ones created with last version, the first chapter title.... Really strange. I'm using Voice https://github.com/PaulWoitaschek/Voice

I'm rendering right now one of the books with the old versión just to see if the text has something to do in this.

I'll keep investigating and I'll tell you.

**Comment by aedocw** [comment]
You can extract the metadata and compare as well, with:
`ffmpeg -i mybook.m4b -loglevel error -f ffmetadata meta.txt`

Maybe compare older one and newer one, and see if you notice anything different? Let me know what you find!

**Comment by friki67** [comment]
Thank you. Nothing has changed. All the metadata is in place.

Well, one thing has changed. The time I added the books to the player. And this is the thing. I suppose that a new version of the player is missmatching things, so I removed all books, moved them to a new folder, and now all of them have the "wrong" title. So I'm going to Voice's github page to report this. Any sugerence for a good Android player?
